app/src/userpages/components/DataUnionPage/Item.jsx

```diff
@@ -378,11 +388,73 @@ const Item = ({ product, stats }: Props) => {
         setIsOpen(!isOpen)
     }, [isOpen])
 
+    const validate = useCallback((product) => {
+        const invalidFields = validateProduct(product, isEthIdentityRequired)
+        const errors = Object.keys(invalidFields)
+            .filter((field) => !!invalidFields[field])
+            .map((field) => field)
+
+        if (errors.length > 0) {
+            errors.forEach((field) => {
+                Notification.push({
+                    title: validationErrors[field],
+                    icon: NotificationIcon.ERROR,
+                })
+            })
+
+            return false
+        }
+
+        if (isEthereumAddress(product.beneficiaryAddress)) {
+            const { active: activeMembers } = (stats && stats.memberCount) || {}
+
+            if ((activeMembers || 0) < dataUnionMemberLimit) {
+                Notification.push({
+                    title: `The minimum community size for a Data Union is ${dataUnionMemberLimit === 1 ?
+                        'one member' :
+                        `${numberToText(dataUnionMemberLimit)} members`
+                    }.`,
+                    icon: NotificationIcon.ERROR,
+                })
+                return false
+            }
+        }
+
+        return true
+    }, [isEthIdentityRequired, stats])
+
+    const publish = useCallback(async (productId) => wrapPublish(async () => {
+        const product = await getProductById(productId || '')
+        const productWithPendingChanges = withPendingChanges(product)
+
+        if (validate(productWithPendingChanges)) {
+            await publishDialog.open({
+                product,
+            })
+        }
+    }), [wrapPublish, validate, publishDialog])
+
+    const productInitials = useMemo(() => {
+        const initials = (productName || '').split(/\s+/).filter(Boolean).map((s) => s[0]).slice(0, 2)
+            .join('')
+            .toUpperCase()
+
+        return (
+            <Initials>
+                {initials}
+            </Initials>
+        )
+    }, [productName])
+
     return (
         <Container>
             <Header>
                 <ImageContainer onClick={onHeaderClick}>
-                    <Image src={product.imageUrl} />
+                    <FallbackImage
```

Comment:
Added product initials as default image, a cover image is required before deploy can be done so not much else to show here